src/udisksdaemonutil.c

```diff
@@ -706,6 +709,7 @@ udisks_daemon_util_check_authorization_sync (UDisksDaemon          *daemon,
                                              GDBusMethodInvocation *invocation)
 {
   GError *error = NULL;
+  #ifdef HAVE_POLKIT_AGENT_1
```

Comment:
This disables all checks if there is no PolicyKit agent. So all users can now use functions that only privileged users should be able to. We have the `check_authorization_no_polkit` which is used if `authority` is `NULL`.